tmpl/formats/epub.go

```diff
@@ -3,31 +3,129 @@ package formats
 import (
 	"archive/zip"
 	"context"
+	"fmt"
+	"github.com/rushsteve1/mangadex-opds/shared"
+	"github.com/rushsteve1/mangadex-opds/tmpl"
+	"golang.org/x/sync/errgroup"
 	"io"
+	"log/slog"
+	"os"
+	"path"
 
 	"github.com/rushsteve1/mangadex-opds/models"
 )
 
 // WriteEpub will write an EPUB file for the current [Chapter] to the given [io.Writer].
 func WriteEpub(ctx context.Context, c *models.Chapter, w io.Writer) (err error) {
-	// A CBZ is almost a valid EPUB so we can use that as the base
-	WriteCBZ(ctx, c, w)
+	z := zip.NewWriter(w)
 
-	// err = c.writeEpubMetadata(z)
+	err = z.SetComment(c.FullTitle())
+	if err != nil {
+		return err
+	}
 
-	return err
-}
+	err = z.AddFS(os.DirFS("tmpl/formats/epub"))
```

Comment:
This reads directly from the filesystem, so will only work when run from the source code folder.

Instead you can use the `embed` module to copy the files directly into the binary, and then use `embed.FS` here